app/assets/javascripts/admin/tasktype/task_type_create_view.js

```diff
@@ -149,7 +149,7 @@ class TaskTypeCreateView extends React.PureComponent<Props, State> {
             <FormItem label="Settings">
               {getFieldDecorator("settings.somaClickingAllowed", {
                 valuePropName: "checked",
-              })(<Checkbox>Allow Single-Node-Tree Mode</Checkbox>)}
+              })(<Checkbox>{`Allow Single-node-tree mode ("Soma clicking")${""}`}</Checkbox>)}
```

Comment:
Did you insert `${""}` to prevent linting errors for using the backticks? You should be able to use double quotes and escape the inner quotes with `&quot;` (if you use single quotes for this string, prettier will automatically replace them with double quotes and do the escaping for the inner double quotes ;)). Same for the other strings.